backend/processing/validation_failure/task.py

```diff
@@ -0,0 +1,9 @@
+"""
+TODO: add docstring here.
+"""
+
+
+def lambda_handler(event, _context):  # pylint:disable=unused-argument
```

Comment:
If you add an underscore to the start of the first argument you shouldn't need the `pylint` annotation.

Reply:
Done in 599f367570528d25a8e23b358e86182fe62eca96